components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/internal/impl/DefaultPermissionProvider.java

```diff
@@ -223,7 +223,11 @@ public boolean hasPermission(String username, Permission permission) throws Perm
     }
 
     public boolean hasPermission(String username, String permissionID) throws PermissionException {
-        return false;
+        if (log.isDebugEnabled()) {
+            log.debug("Check permission " + permissionID);
```

Comment:
@NisalaNiroshana , Provide more context to above debug log. Include username as well to the debug log.